pkg/operation/botanist/component/kubecontrollermanager/waiter.go

```diff
@@ -88,7 +88,7 @@ func (k *kubeControllerManager) WaitForControllerToBeActive(ctx context.Context)
 		}
 
 		// Check if the controller is active by reading its leader election record.
-		lock := resourcelock.EndpointsResourceLock
+		lock := resourcelock.EndpointsLeasesResourceLock
```

Comment:
Instead of using `EndpointsLeasesResourceLock` just don't use any constant (since there is none anymore) and write "endpoints" explicitly here.